src/nucname.h

```diff
@@ -84,7 +84,10 @@ namespace nucname
   {
   public:
     /// default constructor
-    NotANuclide () {};
+    NotANuclide () 
+    {
+      NaNEstr = "";
+    };
```

Comment:
We can use an initializer here:
```suggestion
    NotANuclide () : NaNEstr("") {};
```